testdata/scripts/literals.txt

```diff
@@ -1,13 +1,21 @@
-go run .
+go build
+exec ./main$exe
+binsubstr main$exe 'Lorem' 'dolor' 'second assign' 'First Line' 'Second Line' 'map value' 'to obfuscate' 'also obfuscate' 'stringTypeField String' 
+
+binsubint main$exe '-7081390804778629748' '-301627827188279046' '7679634459002713443'
+binsubfloat main$exe '3684433217126772357.33' '-9015867427900753906'
 cmp stderr main.stderr
 cp stderr normal.stderr
 garble -literals build
 exec ./main$exe
 cmp stderr main.stderr
 cmp stderr normal.stderr
-! binsubstr main$exe 'garbleDecrypt' 'Lorem' 'ipsum' 'dolor' 'first assign' 'second assign' 'First Line' 'Second Line' 'map value' 'to obfuscate' 'also obfuscate' 'stringTypeField String' 'stringTypeStruct'
+! binsubstr main$exe 'garbleDecrypt' 'Lorem' 'dolor' 'first assign' 'second assign' 'First Line' 'Second Line' 'map value' 'to obfuscate' 'also obfuscate' 'stringTypeField String' 
 
-binsubstr main$exe 'also skip this' 'skip typed const' 'skip typed var' 'skip typed var assign' 'stringTypeField strType' 'stringType lambda func return' 'testMap1 key' 'testMap2 key' 'testMap3 key' 'testMap1 value' 'testMap3 value' 'testMap1 new value' 'testMap3 new value' 'stringType func param' 'stringType return' 'skip untyped const'
+binsubstr main$exe 'Skip this block' 'also skip this' 'skip typed const' 'skip typed var' 'skip typed var assign' 'stringTypeField strType' 'stringType lambda func return' 'testMap1 key' 'testMap2 key' 'testMap3 key' 'testMap1 value' 'testMap3 value' 'testMap1 new value' 'testMap3 new value' 'stringType func param' 'stringType return' 'skip untyped const'
+
+! binsubint main$exe '-7081390804778629748' '-301627827188279046' '7679634459002713443'
+! binsubfloat main$exe '3684433217126772357.33' '-9015867427900753906'
```

Comment:
each of these calls has to load the entire binary into memory, so I would suggest to use a single `binsubnumber` command, which accepts both integer and floating point numbers, and searches for them all with a single []byte executable. you could tell which arguments are floats by whether they have a dot in them; so your last argument here could be `-9015867427900753906.0`.

Reply:
Maybe keep the different functions and just cache the binary once its loaded.

Reply:
Be careful with caching, though, because

1) the same binary can change in the middle of a test (e.g. if we go from `garble build` to `go build`)
2) you'll probably be holding onto a large []byte for the entire duration of the test

Reply:
Implement sth similar for binclude already should be easy.